components/about/CustomTable.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+    <div class='officer-table'>
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 spaces but found 4 spaces.](https://app.codacy.com/gh/AIESEC-in-Japan/aiesecjp-web/pullRequest?prid=7665075)
```suggestion
  <div class='officer-table'>
```